SPECS/ipv6calc/ipv6calc.spec

```diff
@@ -0,0 +1,661 @@
+Requires:       httpd
+Requires:       ipv6calc
+Requires:       perl(Digest::MD5)
+
+%description ipv6calcweb
+ipv6calcweb contains a CGI program and a configuration file for
+displaying information of IP addresses on a web page using ipv6calc.
+
+Check/adjust %{_sysconfdir}/httpd/conf.d/ipv6calcweb.conf
+Default restricts access to localhost
+
+%prep
+%if 0%{?gitcommit:1}
+%setup -q -n %{name}-%{gitcommit}
+%else
+%setup -q
+%endif
+
+%patch0 -p1
+
+%configure \
+	%{?enable_ip2location:--enable-ip2location} \
+	%{?enable_ip2location:--with-ip2location-dynamic --with-ip2location-headers-fallback} \
+	--with-ip2location-db=%{ip2location_db} \
+	%{?enable_geoip:--enable-geoip} \
+	%{?enable_geoip:--with-geoip-dynamic --with-geoip-headers-fallback} \
+	--with-geoip-db=%{geoip_db} \
+	%{?enable_dbip:--enable-dbip} \
+	--with-dbip-db=%{dbip_db} \
+	%{?enable_mmdb:--enable-mmdb --with-mmdb-dynamic} \
+	%{?enable_external:--enable-external} \
+	--with-external-db=%{external_db} \
+	%{?enable_shared:--enable-shared} \
+	%{?enable_mod_ipv6calc:--enable-mod_ipv6calc}
+
+
+%build
+make clean
+make %{?_smp_mflags} COPTS="%{optflags}"
+
+
+%install
+make install DESTDIR=%{buildroot}
+
+## Install examples and helper files
+install -d -p %{buildroot}%{_docdir}/%{name}-%{version}/
+
+## examples
+install -d %{buildroot}%{_datadir}/%{name}/examples/
+
+# ipv6logconv
+install -d %{buildroot}%{_datadir}/%{name}/examples/ipv6logconv
+for file in examples/analog/*.{cfg,txt,tab,sh}; do
+	install $file %{buildroot}%{_datadir}/%{name}/examples/ipv6logconv/
+done
+
+# ipv6loganon
+install -d %{buildroot}%{_datadir}/%{name}/examples/ipv6loganon
+for file in ipv6loganon/README; do
+	install $file %{buildroot}%{_datadir}/%{name}/examples/ipv6loganon/
+done
+
+# ipv6logstats
+install -d %{buildroot}%{_datadir}/%{name}/examples/ipv6logstats
+for file in ipv6logstats/README ipv6logstats/example_* ipv6logstats/collect_ipv6logstats.pl; do
+	install $file %{buildroot}%{_datadir}/%{name}/examples/ipv6logstats/
+done
+
+
+# db directory
+install -d %{buildroot}%{external_db}
+
+# selinux
+install -d %{buildroot}%{_datadir}/%{name}/selinux
+
+
+# ipv6calcweb
+install -d %{buildroot}%{_sysconfdir}/httpd/conf.d
+install -d %{buildroot}%{_localstatedir}/www/cgi-bin
+
+install ipv6calcweb/ipv6calcweb.conf %{buildroot}%{_sysconfdir}/httpd/conf.d/
+install -m 755 ipv6calcweb/ipv6calcweb.cgi  %{buildroot}%{_localstatedir}/www/cgi-bin/
+install -m 644 ipv6calcweb/ipv6calcweb-databases-in-var.te  %{buildroot}%{_datadir}/%{name}/selinux/
+
+%clean
+rm -rf %{buildroot}
+
+
+
+%check
+%ifnarch ppc64
+	make test
+%endif
+
+
+%files
+%if %{rpm_license_extra}
+%doc ChangeLog README README.* CREDITS TODO USAGE doc/ipv6calc.lyx doc/ipv6calc.sgml doc/ipv6calc.html doc/ipv6calc.xml
+%license COPYING LICENSE
+%else
+%license COPYING LICENSE
+%doc ChangeLog README README.* CREDITS TODO USAGE doc/ipv6calc.lyx doc/ipv6calc.sgml doc/ipv6calc.html doc/ipv6calc.xml
+%endif
```

Comment:
These look identical. Does changing the order of `%doc` and `%license` make a difference?

Reply:
The linter came in and automatically moved COPYING and LICENSE out of the %doc line, I'll clean it up to just have one.